cli.md

```
@@ -0,0 +1,484 @@
+```bash
+find [検索開始パス] [条件] [アクション]
+```
+
+### 2.2 実践ワンライナー表
+
+| 目的 | ワンライナー | 解説 |
+|---|---|---|
+| 直近60分以内に更新されたファイルを探す | `find . -mmin -60 -type f` | `-mmin -60` は「60分以内に更新」を意味する |
+| 7日以上前の `.log` ファイルを確認する | `find /var/log -name "*.log" -mtime +6` | `-mtime +6` は「更新から7日以上経過」。まずは確認のみで削除しない |
+| 空のディレクトリを一括削除する | `find . -type d -empty -delete` | `-delete` は該当した空ディレクトリのみ削除 |
+| 拡張子ごとのファイル数を集計する | `find . -type f -exec basename {} + | sed -n 's/.*\.\([^.]*\)$/\1/p' | sort | uniq -c | sort -rn` | `find` 結果を `basename` で変換後、拡張子のみ抽出して集計するパイプライン |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)